cmd/hologram-agent/main.go

```diff
@@ -62,10 +64,18 @@ func main() {
 		log.Debug("Using command-line remote address.")
 		config.Host = *dialAddress
 	}
+	if *accessKey != "" {
```

Comment:
Why not leverage the default functionality of aws-sdk-go here? Instead of allowing for cli arguments or putting something into hologram's config.json, just use the out-of-the box default credential provider chain of your library!

https://github.com/aws/aws-sdk-go/#configuring-credentials

Or am I missing a reason for not leveraging that?